src/main/java/frc/robot/OI.java

```diff
@@ -174,6 +163,11 @@ private void setButtonAction(String name){
         private static final Button kBuddyBalanceActivateButton = Button.B; //Activates buddy balance
         private static final Button kDownBuddyBalanceButton = Button.POVDOWN; // Lowers buddy balance lift
         private static final Button kUpBuddyBalanceButton = Button.POVUP; // Raises buddy balance lift
+        private static final Button kYellowCargo = Button.Y;
+        private static final Button kPurpleCargo = Button.X;
+
+        private static final int kArmElow = 0;
```

Comment:
typo: Elow instead of Elbow

Should we name it kArmElbowAxis?  Same below for base.